.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Task
+about: Add an issue to describe and track an implementation task
+title: ''
+labels: kind/task
```

Comment:
I think that we need a template for issues under a user story epic, so added this.  Will need to add this label separately, if PR is approved.

Reply:
I just figure this is worth asking. is anything lost by moving to `kind/task` vs `kind/enhancement` + some other `kind/X` cards for the tasks?

Reply:
I actually don't have a strong opinion.  Mostly, I would love to just have a template for this, since it feels like such a common type of issue (i.e. something under a user story).  If we didn't use `kind/task`, I would imagine that we would want to use a variety of labels based on the issue, i.e. `kind/enhancement`, `kind/tech-debt`, `kind/bug`.

Reply:
maybe just drop `kind/task` for now?

Reply:
I like to be able to measure time spent on different types of work (e.g. enhancements/PM driven work, tech debt, bugs, support, etc.) I think having kind/task would interfere with that (although I'm not sure how the kibana metrics treat issues with multiple labels. I agree that we need a separate template for tasks though.

Reply:
Can we keep this as the template we use for tasks but drop the auto-add of a `kind/task` label?
```suggestion
```

Reply:
if we can agree on this change, I'm ready to approve this and we can evolve it further as we go

Reply:
Sure, thats fine with me.

Reply:
Actually, I deleted the file and rebased, so I think it's not easily retrieved.  Lets just get this merged and it can be easy to add a task in the future.